lib/ain-contracts/src/lib.rs

```diff
@@ -103,20 +103,60 @@ pub struct FixedContract {
 }
 
 lazy_static::lazy_static! {
-    pub static ref INTRINSIC_CONTRACT: FixedContract = {
-        let bytecode = solc_artifact_bytecode_str!("dfi_intrinsics", "deployed_bytecode.json");
+    pub static ref IMPLEMENTATION_SLOT: H256 = H256::from_str("0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc").unwrap();
```

Comment:
Can drop the unwrap and pass the [u8; 32] array here. Can then be lifted out of lazy_static

Reply:
Fixed: 78ffbf4